tf-101/1-why-iac.md

```diff
@@ -25,51 +25,74 @@ redirect: https://developer.oracle.com/tutorials/tf-101/1-why-iac/
 
-## So Many Ways to Manage
+This tutorial will cover many of the common ways to manage IT resources, explore the upsides and downsides of each, and get you a little more familiar with IaC.
 
-Let's start with the basics.  When there are IT resources to manage, what are some of our common options?
+For additional information, see:
 
-* GUI (aka Console)
-* Command-Line Interface (CLI)
+* [Signing Up for Oracle Cloud Infrastructure](https://docs.oracle.com/iaas/Content/GSG/Tasks/signingup.htm)
+* [About the User Interfaces of the Oracle Integration Oracle Cloud Infrastructure (OCI) Console](https://docs.oracle.com/en/cloud/paas/integration-cloud/integration-cloud-auton/user-interfaces-oracle-integration-cloud-1.html)
+* [Working with the Command Line Interface (CLI)](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/cliconcepts.htm)
+* [IaC tools](https://cool.devo.build/topics/iac)
+
+## So many Ways to manage
+
+Let's start with the basics. When there are IT resources to manage, what are some of the common options?
+
+* GUI (aka Oracle Cloud Infrastructure (OCI) Console)
+* Command Line Interface (CLI)
 * API
 * IaC tools
 
 ### GUIs
 
-Most of us start with the GUI.  In Oracle Cloud Infrastructure (OCI) this is the [OCI Console](https://www.oracle.com/cloud/sign-in.html).  GUIs are typically pretty fancy, fun, and easy to use.  While they can be entertaining and great for initially learning a system/platform, they're not always the most scalable or efficient.  UIs require time and manual user intervention (unless you're using something to automate the pointing-and-clicking, such as [Selenium](https://www.selenium.dev), but that's a pretty niche edge case).  Things only "work" when someone's there to click a button.  It's difficult (read: *impossible*) to rapidly manage resources using a UI.  Just the time it takes to point-and-click, plus wait for the browser to update... it's not super fast.
+In many applications, most of us start with the GUI. If you work within the Oracle Cloud Infrastructure (OCI), you're probably already familiar with the [OCI Console](https://www.oracle.com/cloud/sign-in.html). Overally, GUIs are typically pretty fancy, fun, and easy to use. But while they can be entertaining and great for initially learning a system or platform, they're not always the most scalable or efficient. Also, GUIs require time and manual user intervention, unless you're using something to automate the pointing-and-clicking, such as [Selenium](https://www.selenium.dev) (a fairly niche edge case). Things only "work" when someone's there to click a button. It's difficult (read: *impossible*) to rapidly manage resources using a GUI. Just the time it takes to point, click, and wait for the browser to update isn't really ideal when you're trying to handle a critical situation.
```

Comment:
Overally?

Reply:
Nope. :) I think part of my brain was going with 'really' and just combined the two. Also, I may need to look for a dictionary extension, since VSC isn't flagging errors of that nature.